src/js/utils/background.js

```diff
@@ -159,8 +165,16 @@ export const backgroundAndTextColors = (backgroundArg, textArg, theme) => {
   } else {
     backgroundColor = normalizeBackgroundColor(background, theme);
     const shade = darkContext(backgroundColor, theme);
+    let transparent;
+
+    if (backgroundColor && canExtractRGBArray(backgroundColor)) {
+      const colorArray = getRGBArray(backgroundColor);
+      if (colorArray[3] < 0.5) transparent = true;
```

Comment:
add a comment that `[3]` refers to the alpha value?

Reply:
I added a comment in the latest commit